Fix procedure about configuring LMS #6273 (#6427)
github/docs: content/education/manage-coursework-with-github-classroom/manage-classrooms.md

{% data reusables.classroom.click-students %}
1. To connect {% data variables.product.prodname_classroom %} to your LMS and import a roster, click {% octicon "mortar-board" aria-label="The mortar board icon" %} **Import from a learning management system** and follow the instructions. For more information, see "[Connect a learning management system to {% data variables.product.prodname_classroom %}](/education/manage-coursework-with-github-classroom/connect-a-learning-management-system-to-github-classroom)."
    !["Import from a learning management system" button](/assets/images/help/classroom/click-import-from-a-learning-management-system-button.png)

> 1. Create your class roster.

Reviewer: ```suggestion
1. Provide the student identifiers for your roster.
```

> 1. Choose whether to import a roster or create a roster manually.

Reviewer: For a step like this, with two-sub steps, we want the original step to be a summary of all sub-steps. Choosing whether you want to import a roster or creating one manually is really a different step than doing the actual importing. However, "create your roster" is a good summary, and the user can choose to accomplish the step either by importing or typing. 😊 

```suggestion
1. Create your class roster.
```

>     - To import a roster, upload a file containing the identifiers, click **Upload a CSV or text file**.

Reviewer: ```suggestion
    - To import a roster by uploading a file containing student identifiers, click **Upload a CSV or text file**.
```

>     - Otherwise, to create a roster manually, type your student identifiers.

Reviewer: ```suggestion
    - To create a roster manually, type your student identifiers.
```
